recipes/lammps/2025/eiger/extra/reframe.yaml

```
@@ -0,0 +1,19 @@
+build-kokkos-eiger:
+  features:
+    - mpi
+    - lammps-kokkos-dev-eiger
```

```suggestion
    - lammps-kokkos-dev
```
? Shouldn't reframe detect which system it's running on instead?

---

yeah you're right - i've fixed this in latest commit.